scripts/tests/chiptest/test_definition.py

```diff
@@ -372,11 +375,19 @@ def Run(self, runner, apps_register, paths: ApplicationPaths, pics_file: str,
                     runner.RunSubprocess(python_cmd, name='CHIP_REPL_YAML_TESTER',
                                          dependencies=[apps_register], timeout_seconds=timeout_seconds)
             else:
-                pairing_cmd = paths.chip_tool_with_python_cmd + ['pairing', 'code', TEST_NODE_ID, setupCode]
+                pairing_server_args = []
+                if tool_hci_number is not None:
+                    pairing_cmd = paths.chip_tool_with_python_cmd + [
+                        "pairing", "ble-wifi",  TEST_NODE_ID, "Virtual_Wifi", "ExamplePassword", "20202021",  "3840", ]
+                    pairing_server_args = [
+                        "--ble-adapter", str(tool_hci_number)]
```

Comment:
`pairing code-wifi` should work, no?  Why is this using `ble-wifi`, which is not really maintained?

Reply:
@bzbarsky-apple 
At that time I wasn't aware of cmd `pairing code-wifi`. I've change implementation to use new cmd and update everything to works with ubuntu 24.04